src/main/java/spongebob/task/Event.java

```diff
@@ -0,0 +1,80 @@
+package spongebob.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import spongebob.exception.SpongebobException;
+
+
+/**
+ * An Event task, contains a dateTime for start and enddate
+ */
+public class Event extends Task {
+
+    protected LocalDate from;
+    protected LocalDate to;
+
+    /**
+     * constructor for an event task, checks if description, start and end dates to be in correct format
```

Comment:
In method header comments, the first sentence should start with a verb. Can be:  Construct an instance of Event after checking the format of description, start and end dates.

Reply:
Good point, i will update the description!